### PR TITLE
Migrate README from rdoc to markdown

### PR DESCRIPTION
**Pull request by cdpalmer**
# Summary
Migrate README to markdown to provide syntax highlighting

# Checklist
- [x] Ran unit tests (`rake`)

# Note: 
This automated checklist may be outdated.  When I run `rake`, it runs all the integration specs as well.  I had to specifically run: `rake test:unit`.  I'll create an issue.

### CODE REVIEW

File: CHANGELOG.rdoc [hunk 1]
```diff
@@ -1,3 +1,6 @@
+== 2.80.2
```
Comment (crookedneighbor) [inline CHANGELOG.rdoc]: Please set this to unreleased. We should update the Pull Request template to make that clearer.

Comment (crookedneighbor) [reply CHANGELOG.rdoc]: Also, I don't know that this kind of change even merits a Changelog entry.

Comment (cdpalmer) [reply CHANGELOG.rdoc]: Absolutely.  I wasn't sure what merited a CHANGELOG entry for you guys.  Will change

**Comment by cdpalmer** [comment]
Good question on the ruby gems markdown support.  I have always done my `README` in markdown instead of `rdoc`.  I can look into it

File: CHANGELOG.rdoc (hunk 1 above)
Comment (cdpalmer) [reply CHANGELOG.rdoc]: Fixed.  I also verified that many other big name gems use markdown as well for their README, so I don't think there are negative consequences to using this.

If you list out the criteria for what needs a CHANGELOG entry, I can update your PR template in another branch if you'd like.

**Comment by crookedneighbor** [comment]
Should the changelog be converted to markdown as well? Or does it need to remain an rdoc?

**Comment by cdpalmer** [comment]
I would say there is no need I can see.  The main reason for the conversion is so that when viewing the code on Github, you can have syntax highlighting for the code snippets your provide.  CHANGELOG is pretty basic.

**Comment by crookedneighbor** [comment]
Yeah, I get that. I'm just fuzzy on what the requirements for Ruby gems are.

**Comment by cdpalmer** [comment]
You know, I'm not sure what requirements exist. I looked at rubygems.org instructions for building gems and couldn't find anything explicit.  I looked at the code bases for `rails`, `nokogiri`, and `pry`, and all their README's are markdown files.  So that makes me _think_ there aren't any requirements around that.  But that's the most I could deduce.

**Comment by crookedneighbor** [comment]
Made some slight adjustments and confirmed that the rdoc format is not needed for Ruby Gems.

**Pull request merged**